just-install.json

```diff
@@ -1270,6 +1270,20 @@
       },
       "version": "5.18.1.0"
     },
+    "moonlight": {
```

Comment:
I would rename this `moonlight-stream` or `moonlight-qt`. Moonlight by itself reminds me of [this](https://www.mono-project.com/docs/web/moonlight/) (which is dead, but it is still the first thing that comes to my mind when I read `moonlight`).

Reply:
Absolutely understandable. I'll rename it to moonlight-qt as that's the name of the installer package.